sphinx/source/start_network.rst

```diff
@@ -95,6 +95,34 @@ Other members are then allowed to vote for the proposal, using the proposal ID r
 
 Once a quorum of members have approved the network opening (``"result":true``), the network is opened to users (see :ref:`Example App` for a simple business logic and :term:`JSON-RPC` transactions). It is only then that users are able to execute transactions on the business logic defined by the enclave file (``--enclave-file`` option to ``cchost``).
 
+Add new nodes to an open network
+--------------------------------
+
+Once the network has been opened by members, it is possible to add new nodes to the network (e.g. to replace a retired node or add a new version of the code). The new node should be started with the ``join`` option:
+
+.. code-block:: bash
+
+    $ cchost --enclave-file /path/to/enclave_library --enclave-type debug
+    --node-address node_ip:node_port --rpc-address rpc_ip:rpc_port
+    --public-rpc-address public_rpc_ip:public_rpc_port --ledger-file /path/to/ledger
+    --node-cert-file /path/to/node_certificate --quote-file /path/to/quote
+    join --network-cert-file /path/to/existing/network_certificate --target-rpc-address target_rpc_ip:target_rpc_port
+
+As opposed to an opening network in which nodes are trusted automatically (see :ref:`Add a new node to an opening network`), new nodes added to an open network should be trusted by a quorum of members before becoming part of the network. When a new node joins an open network, it is assigned a unique node id and is recorded in state `PENDING`. Then, members can vote to accept the new node:
+
+.. code-block:: bash
+
+    $ memberclient --cert member1_cert --privk member1_privk --rpc-address rpc_ip:rpc_port --ca network_cert accept_node --node-id new_node_id
```

Comment:
This subcommand should now be called `propose_node` rather than `accept_node`?

Reply:
As discussed, we'll go with `trust_node`.